### PR TITLE
early file creation order

### PR DESCRIPTION
**Pull request by xdel**

### CODE REVIEW

**Comment by aglarendil** [comment]
I suggest you to update all ocf files  orders

**Comment by xdel** [comment]
That`s all, other already have an ordering option.

**Comment by xdel** [comment]
tested on both HAs

**Comment by monroid** [comment]
retest this please

**Comment by xenolog** [comment]
it's  wrong  goose!!!

It cant work, because corosync and pacemaker starts on early stages. And relationship between resources from another stages -- impossible by Puppet design.

**Comment by monroid** [comment]
retest this please

**Comment by monroid** [comment]
retest this please

**Comment by monroid** [comment]
retest this please

File: deployment/puppet/openstack/manifests/corosync.pp [hunk 1]
```diff
@@ -39,15 +39,21 @@
 } 
 File['filter_quantum_ports.py'] -> File<| title == 'quantum-ovs-agent' |>
 
+exec { "create_ocf_dirs":
+    command => "mkdir -p /usr/lib/ocf/resource.d/pacemaker/ /usr/lib/ocf/resource.d/mirantis/",
+    path    => "/usr/bin/:/bin/",
+}
+
+Exec['create_ocf_dirs'] ->
```
Comment (xenolog) [inline deployment/puppet/openstack/manifests/corosync.pp:47]: Why you create directories by exec???
It's not good idea.

**Comment by mihgen** [comment]
Closed because Sergey Vasilenko has another implementation of this. Talk to xdeller for details.

**Pull request closed without merging**